pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "ukrr-models"
+version = "1.1.0"
+description = "Models of the UKRR database"
+authors = ["UK Renal Registry"]
+license = "MIT"
```

Comment:
Outside what I think is the intended scope of this PR but we could do with having a review of licencing etc. and whether we're doing it consistently, appropriately and correctly.

In this instance if we are applying a MIT licence then I think we should also be supplying a LICENCE file in the repo root etc.

Reply:
Yes, the lack of licence is an oversite on my part. Maybe we should have a discussion about this because if I am honest I always slap an MIT on everything for no other reason than that it's better to have something than nothing.

Reply:
I have added a licence and add LICENCE to the list of files to check for